README.md

```diff
@@ -1,8 +1,55 @@
 # OpenTelemetry Collector Contrib
 This is a repository for OpenTelemetry Collector contributions that are not part of the
-core repository and core distribution of the Collector.
+[core repository](https://github.com/open-telemetry/opentelemetry-collector) and
+core distribution of the Collector. Typically, these contributions are vendor
+specific receivers/exporters and/or components that are either legacy or are only
```

Comment:
Nit: do we need to specifically mention "legacy" here? I think this sentence reads fine and serves the purpose without that reference.